### DISCUSSION TITLE
Feature Request: Integrate Obsidian dictionary into default style

### DISCUSSION THREAD

**Discussion by fruchtblase** [marcusolsson/obsidian-vale#14] [Ideas]
Thank you for making this plugin, it seems really useful!

When trying it out, the suggestion list was polluted with a lot of special terms, that I already added to the obsidian dictionary. After manually copying them into the default style, the plugin became usable for me.

I think this plugin could really benefit from an option that did this automatically. (Apologies if this is already planned)

**Comment by marcusolsson** [comment]
That's a cool idea! I have planned to add support for managing vocabs, but I hadn't thought of syncing it with the built-in one. I'm not sure how I can access the dictionary, but if there's a way, I think it makes total sense.

**Comment by fruchtblase** [comment]
I have no idea if plugins can access the file, but the user added words are saved inside the app.json file in the .obsidian folder.

**Comment by dylan-k** [comment]
json converts nicely to yaml, the format used by vale's vocab, but how to keep them in sync I wonder?
